countnumbers-inatext.py

```diff
@@ -0,0 +1,18 @@
+# Python program to count the
+# number of lines in a text file
+
+
+# Opening a file
+file = open("gfg.txt","r")
```

Comment:
sorry , but the code of another platform not considered here..
I am closing this PR.
please create a new PR with a unique code..